gui/default/syncthing/settings/settingsModalView.html

```diff
@@ -111,6 +111,10 @@
         <div id="settings-gui" class="tab-pane">
           <div class="form-group"  ng-class="{'has-error': settingsEditor.Address.$invalid && settingsEditor.Address.$dirty}">
             <label translate for="Address">GUI Listen Address</label>&emsp;<a href="https://docs.syncthing.net/users/guilisten.html" target="_blank"><span class="fas fa-question-circle"></span>&nbsp;<span translate>Help</span></a>
+            <p class="text-warning" ng-show="system.guiAddressOverridden">
+                <span class="fas fa-exclamation-triangle"></span>
+                <span translate>The GUI address is overridden by startup options. Changes here will not take effect.</span>
```

Comment:
Maybe something like "Changes will not take effect as long as these options are in effect."?

Reply:
Agreed, it should be clear that the changes will be saved, and will take effect if the GUI address override is removed.